TimeCapsuleManagement/models.py

```diff
@@ -1,3 +1,57 @@
 from django.db import models
+# from django.contrib.auth.models import User
+from AuthenticationSystem.models import UserProfile
+from django.utils import timezone
 
 # Create your models here.
+
+
+class Capsule(models.Model):
+    owner = models.ForeignKey(UserProfile, related_name='capsules', on_delete=models.CASCADE)
+    name = models.CharField(max_length=100)
+    description = models.TextField(blank=True, null=True)
+    creation_date = models.DateTimeField(auto_now_add=True)
+    last_edited_date = models.DateTimeField(auto_now=True)
+    is_public = models.BooleanField(default=False)
+    unsealing_date = models.DateTimeField()
+
+    def is_unsealed(self):
+        return timezone.now() >= self.unsealing_date
+
+    def __str__(self):
+        return self.name
+
+
+class CapsuleContent(models.Model):
+    FILE_TYPES = (
+        ('photo', 'Photo'),
+        ('video', 'Video'),
+        ('text', 'Text'),
+    )
+    file_type = models.CharField(max_length=20, choices=FILE_TYPES)
+    file = models.FileField(upload_to='capsule_content/%Y/%m/%d/')
```

Comment:
Keep this field as a public url, that should work.

Reply:
Its set up to store the image on your local device. When we set up s3 we can change the path